src/charm.py

```diff
@@ -7,7 +7,7 @@
 # pylint: disable=too-many-lines
```

Comment:
Can you check if we can remove this lint ignore, please?

Reply:
it goes away nicely!